core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/components/dot-favorite-page/dot-favorite-page.component.ts

```diff
@@ -71,7 +71,7 @@ export class DotFavoritePageComponent implements OnInit, OnDestroy {
                         inode: [formStateData?.inode],
                         thumbnail: [formStateData?.thumbnail, Validators.required],
                         title: [formStateData?.title, Validators.required],
-                        url: [{ value: formStateData?.url, disabled: true }, Validators.required],
+                        url: [formStateData?.url, Validators.required],
                         order: [formStateData?.order, Validators.required],
                         permissions: [formStateData?.permissions]
                     });
```

Comment:
Why the `this.isFormValid$` is inside this subscribe? Looks like it doesn't need anything from this subscribe.

Reply:
or use take 1.

Reply:
it depends on the variable `this.form` which is defined only after the subscribe is done